docs/ref/contrib/gis/db-api.txt

```diff
@@ -374,38 +374,38 @@ Database functions
 The following table provides a summary of what geography-specific database
 functions are available on each spatial backend.
 
-====================================  =======  ======  ===========  ==========
-Function                              PostGIS  Oracle  MySQL        SpatiaLite
-====================================  =======  ======  ===========  ==========
-:class:`Area`                         X        X       X            X
-:class:`AsGeoJSON`                    X                             X
-:class:`AsGML`                        X        X                    X
-:class:`AsKML`                        X                             X
-:class:`AsSVG`                        X                             X
-:class:`BoundingCircle`               X
-:class:`Centroid`                     X        X       X            X
-:class:`Difference`                   X        X       X (≥ 5.6.1)  X
-:class:`Distance`                     X        X       X (≥ 5.6.1)  X
-:class:`Envelope`                     X                X            X
+====================================  =======  ==============  ===========  ==========
+Function                              PostGIS  Oracle          MySQL        SpatiaLite
+====================================  =======  ==============  ===========  ==========
+:class:`Area`                         X                        X            X
+:class:`AsGeoJSON`                    X                                     X
+:class:`AsGML`                        X                                     X
+:class:`AsKML`                        X                                     X
+:class:`AsSVG`                        X                                     X
+:class:`BoundingCircle`               X        X (≥ 12.1.0.2)
```

Comment:
Probably we could make columns have equal width. I think `len('X (≥ 12.1.0.2)'` width ought to be enough for any column.